components/menus/OkUserMenu.vue

```diff
@@ -256,6 +258,16 @@
             });
         }
 
+        async onFollowersClick() {
+            this.$emit("leaveMenu");
```

Comment:
What is this event for? Is it something someone else is subscribing to as in` <component @leaveMenu="aFunction"` ? If so, should be `onWantsToLeaveMenu` because by itself it doesnt actually leaves the menu.

Reply:
The same thing is done in `onThemesClick` and `onWantsToGetApp`.